pkg/exec_unit/plugin_assets.go

```diff
@@ -75,6 +84,45 @@ type assetPathMatcher struct {
 	err     error
 }
 
+func (m *assetPathMatcher) ModifyPathsForAnnotatedFile(path string) error {
+	newInclude := []string{}
+	for _, pattern := range m.include {
+		absPath, err := filepath.Abs(pattern)
```

Comment:
Seems odd to convert the pattern to an absolute path, perhaps `IsAbs` would be better?

Also, do we expect these paths to be standard unix or platform-specific? These calls will all be platform specific. If platform specific is intended, the `TrimPrefix` later on the leading slash won't work on Windows.

Reply:
IsAbs is probably better

so youre saying we will see the path for our files as `static/file` on mac or linux and `/static/file` on windows?

If so then why dont we convert everything to an absolute path from the klotho root in readdir so we have a standard path we can follow? Our internal compiler logic shouldnt be dependent on the platform we are running on.